.github/workflows/ci.yml

```diff
@@ -24,6 +26,7 @@ jobs:
       - name: Run chart-testing (list-changed)
         id: list-changed
         run: |
+          ct list-changed --config ct.yml
```

Comment:
Did you mean to leave this line here?

Reply:
no sorry, I was testing. I will remove it. thanks.